src/index.html

```diff
@@ -13,6 +13,10 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div
+      class="calendar calendar--start-day-sun calendar--month-length-31"
```

Comment:
The class names `calendar--start-day-sun` and `calendar--month-length-31` are not following BEM naming convention. According to BEM, modifiers should be separated by a single underscore (_), not a hyphen (-). So it should be `calendar--start_day_sun` and `calendar--month_length_31`.